routers/auth.py

```
@@ -1,32 +1,29 @@
-from fastapi import APIRouter, HTTPException
-from schemas.user import UserLogin
+from fastapi import APIRouter, HTTPException, Depends
+from fastapi.security import OAuth2PasswordRequestForm
 from utils.auth import verify_password, create_access_token
 from utils.data import load_data
 
 router = APIRouter(prefix="/auth",tags=["Auth"])
 
 @router.post("/tokens")
-def login(data: UserLogin):
+def login(form_data: OAuth2PasswordRequestForm = Depends()):
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)